randnum.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
```

Comment:
Overall, decent job - like I said, you could have implemented it more efficiently, but you got the assignment done correctly!